azure-pipelines.yml

```diff
@@ -255,7 +255,7 @@ jobs:
 
           if is_release_commit; then
               echo "##vso[task.setvariable variable=is_release;isOutput=true]true"
-              echo "##vso[task.setvariable variable=trigger_sha;isOutput=true]$(git rev-parse HEAD)"
+              echo "##vso[task.setvariable variable=trigger_sha;isOutput=true]$(branch_sha)"
```

Comment:
Note: `branch_sha` is defined as the current commit when running from master, and as the branch sha (as opposed to the merge sha) on a PR.